tools/parity_scale_decoder/build.rs

```diff
@@ -23,6 +23,8 @@ where
     path_to.push("samples/");
     path_to.push(filename);
 
+    println!("cargo:rerun-if-changed={}", path_to.to_str().unwrap());
```

Comment:
What is the idea behind that? `path_to` will have no file extension here, so no file should be matched

Reply:
Oh, that's definitely a mistake on my part!

It was supposed to be a path to the json file. I had to edit one in this PR, but the `build.rs` was not getting re-ran so I figured I would do a drive-by fix.

Reply:
should be fixed now